src/common/utils/chart.tsx

```diff
@@ -0,0 +1,18 @@
+import { Level } from 'common/level';
+
+/**
+ * Depending on provided `level`, returns the provided `lowText`, `mediumText`,
+ * or `highText`.
+ */
+export function levelText(
+  level: Level,
+  lowText: string,
+  mediumText: string,
+  highText: string,
+) {
+  return level === Level.LOW
+    ? lowText
+    : level === Level.MEDIUM
+    ? mediumText
+    : highText;
+}
```

Comment:
Non-blocking: I'd recommend a higher order function here. I imagine something like this:
```typescript
function textLevelerGenerator(): (level: string) => string {};

const textLeveler = textLevelerGenerator({ lowText: 'low', mediumText: 'medium', highText: 'high' });

// At call site
const leveledText = textLeveler(level);

```

I think at least making explicit what the first 3 args are by using an object will be clearer when you're calling the function.